### PR TITLE
feat: better search

### PR DESCRIPTION
**Pull request by gabrielcolson**
- remove nexus-plugin-prisma (deprecated)
- upgrade prisma to v3.5.0
- improve search results and relevance (thanks to Postgres full text search)
- fix: go to page 1 when submitting a new search in front

### CODE REVIEW

File: back/src/graphql/index.ts [hunk 1]
```diff
@@ -23,6 +23,7 @@ function formatError(err: GraphQLError): Error {
     return new UserInputError(originalError.message);
   }
 
+  console.error('Internal server error:', originalError);
```
Comment (chloebourbion) [inline back/src/graphql/index.ts:27]: do you need to keep this?

File: back/src/modules/packages/packages.service.ts [hunk 2]
```diff
@@ -39,25 +39,46 @@ export async function search(s: string): Promise<Package[]> {
 
   const keyword = s.replace(/[a-zA-Z-]+:[a-zA-Z-]+/g, '').trim();
 
+  let orderBy = sortKeys[parameters.sort ?? 'downloads-desc'];
```
Comment (chloebourbion) [inline back/src/modules/packages/packages.service.ts:42]: can you type parameters ?

File: back/src/graphql/index.ts (hunk 1 above)
Comment (gabrielcolson) [reply back/src/graphql/index.ts:27]: Yes ! It ads logs in case of an unknown error

**Review by chloebourbion — APPROVED** [review]
It works well! The search by word/letter is better
Please fix the lint so we can merge

**Pull request merged**